.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
+      id: go
+
+    - name: Set up protoc
+      uses: arduino/setup-protoc@v1.1.0
+      with:
+        version: 3.x
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Initialize submodules
+      run: git submodule update --init
+
+    - name: Runs tests
+      run: make
```

Comment:
I think we should also run `make integration_test` here (as we used to do on Travis)

Reply:
I didn't realize we used to have a Travis setup. I'll add `make integration_test`--we might need additional Docker setup for that, but there should be existing actions to support that.

I also see a `deploy` configuration in `.travis.yml`--is that something we should use actions for as well?

Reply:
Yeah, we should also move the deploy configuration, but lets do that separately - its what's responsible for creating a binary in the releases list.